### PR TITLE
Update of scala version and libraries versions

### PR DESCRIPTION
**Pull request by alex1712**
Feel free to release it if you like, the tests are green but I didn't have use the new version yet.

Thanks.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -2,14 +2,15 @@ organization := "org.stanch"
 
 name := "play-json-zipper"
 
-version := "1.5"
+version := "2.0"
 
-scalaVersion := "2.11.8"
+scalaVersion := "2.12.2"
```
Comment (stanch) [inline build.sbt:7]: Could you add a cross-build for 2.11?

Comment (alex1712) [reply build.sbt:7]: Done :)

Comment (stanch) [reply build.sbt:7]: Sorry to be a PITA, but could you change to `2.11.11`? :)

Comment (alex1712) [reply build.sbt:7]: I put the version which was in the project. Now updated to 2.11.11 thanks.

**Comment by stanch** [comment]
Awesome, thanks! Just cross-published `2.0`.

**Pull request merged**